tests/plugins/table/headers1row1col.html

```diff
@@ -0,0 +1,40 @@
+<textarea id="none">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<tbody>
+			<tr>
+				<td>^cell 1.1</td>
+			</tr>
+		</tbody>
+	</table>
+</textarea>
+<textarea id="col">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<tbody>
+			<tr>
+				<th scope="row">^cell 1.1</th>
+			</tr>
+		</tbody>
+	</table>
+</textarea>
+<textarea id="row">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<thead>
+			<tr>
+				<th scope="col">^cell 1.1</th>
+			</tr>
+		</thead>
+		<tbody>
+		</tbody>
+	</table>
+</textarea>
+<textarea id="both">
```

Comment:
Isn't `row` and `both` html the same? If so there is no need to duplicate it (and also no need to duplicate test cases).

Reply:
Maybe to make clear that one fixture is used for both `row` and `both` cases, `id="both"` could be changed to `class="row both"`, so when looking for element in TC selector would match actual searched case. Otherwise it might be a bit inconvenient if fixture named `both` is uses for `row` case.

Reply:
I did as @engineering-this suggested - I changed `id` attribute to `class` and reused table code clearly indicating to which test cases it belongs.